docs/components/data-tile.md

```diff
@@ -29,12 +29,11 @@ eleventyNavigation:
   </div>
 </div>
 
-
 ```html
 <div class="data-tile">
   <div class="data-tile-body">
     <p class="fw-bold text-black">Label</p>
-    <hr class="minor">
+    <hr class="minor" />
```

Comment:
It looks like it doesn't look at the njk files, so probably something we'll need to add for prettier, but yeah, it does try to format and fix code inside of code blocks if it can determine the language.